README.md

```diff
@@ -283,6 +295,34 @@ If you would like to cite the FIRST methodology, please consider citing:
 }
 ```
 
+If you woud like to cite the ListT5's tournament sort methodology, please consider citing
```

Comment:
I don't think this is the first instance of tournament sort, something has probably been done before, maybe we drop it for now